src/main/java/com/gd/clinic/entities/Doctor.java

```diff
@@ -0,0 +1,29 @@
+package com.gd.clinic.entities;
+
+import com.sun.xml.bind.v2.TODO;
```

Comment:
this is not needed

Reply:
Sure, was trying to get a `TODO` annotation in the comments and that got imported, didn't notice it stayed there. Removed the import statement this commit.